lib/session/call-session.js

```diff
@@ -2033,8 +2034,12 @@ Duration=${duration} `
   }
 
   _configMsEndpoint() {
```

Comment:
https://github.com/jambonz/jambonz-feature-server/blob/main/lib/utils/place-outdial.js#L326

We might want to update this.

Reply:
we basically have two duplicate functions?  Is there no way to centralize that code?